crates/unparser/test_files/simple_comprehensions.py

```
@@ -0,0 +1,12 @@
+import random
+import string
+
+generator = (value for value in random.choices(string.ascii_letters))
+set_ = {def_ for def_ in random.choices(string.ascii_letters)}
+dict_ = {k: v for k, v in enumerate(random.choices(string.ascii_letters))}
+list_ = [value for value in random.choices(string.ascii_letters)]
+
+
+with_outer_if = (value for value in random.choices(string.ascii_letters) if value)
+with_outer_if_not = (value for value in random.choices(string.ascii_letters) if not value if value != value)
```

Ruff (0.11.9)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)